resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "message": "The LISA account is already closed"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing a LISA account which has already been cancelled</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "2000000403",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CANCELLED",
+          "message": "The LISA account is already cancelled"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing a LISA account which has already been voided</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "1000000403",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_VOID",
+          "message": "The LISA account is already void"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request with an invalid 'Accept' header</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a><br><br>Accept: application/vnd.hmrc.1.0</p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543210",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for a pre-existing account</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks